src/main/webapp/resources/js/apis/projects/project.js

```diff
@@ -88,3 +88,15 @@ export const {
   useUpdateProjectPriorityMutation,
   useUpdateDefaultMetadataTemplateMutation,
 } = projectApi;
+
+export async function deleteProject(projectId) {
+  try {
+    await fetch(`${BASE_URL}/delete?projectId=${projectId}`, {
+      method: "POST",
```

Comment:
As mentioned above for the controller can we change this to a Delete? And can't we add this to the above code?

Reply:
Fixed